statistics-dashboard-common/src/main/java/statistics/dashboard/common/view/ResultListFilters.java

```diff
@@ -0,0 +1,24 @@
+package statistics.dashboard.common.view;
+
+import java.util.List;
+
+/**
+ * A wrapper class that saves all the breakdowns results
+ */
+
+public class ResultListFilters {
+
+  private List<BreakdownResult> allBreakdowns;
+
+  public ResultListFilters(List<BreakdownResult> allBreakdowns) {
+    this.allBreakdowns = allBreakdowns;
```

Comment:
This goes for all request and response classes. Wherever you set a list, it is good practice to copy it (so that the caller cannot mess with the list inside this object).

Reply:
Done